.github/workflows/integration-tests.yml

```diff
@@ -461,14 +463,16 @@ jobs:
       ## Run this step when changes that require tests to be run are made
       - name: Run Tests
         if: needs.changes.outputs.src == 'true'
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@7882cf348cd6a1f6bcf1ee8280185584ebba96e9 # v2.3.10
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@5dd916d08c03cb5f9a97304f4f174820421bb946 # v2.3.11
         with:
-          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json -test.parallel=${{ matrix.product.nodes }} ${{ steps.build-go-test-command.outputs.run_command }} 2>&1 | tee /tmp/gotest.log | gotestfmt
+          test_command_to_run: cd ./integration-tests && go test -timeout 30m -count=1 -json -test.parallel=${{ matrix.product.nodes }} ${{ steps.build-go-test-command.outputs.run_command }} 2>&1 | tee /tmp/gotest.log | gotestloghelper -ci -singlepackage
```

Comment:
how about merging` tee /tmp/gotest.log | gotestloghelper -ci -singlepackage` to `test_command_to_run` if opted for `gotestloghelper` in run-test? Might help in maintenance and updating all run-tests. Can take an input to denote if gotesthelper is preferred.

Reply:
That is a good idea. I will be ripping out the run-tests from chainlink-github-actions and moving it to .github soon so I will add this to the changes that would improve the flow there!